constantine/math/elliptic/ec_twistededwards_batch_ops.nim

```diff
@@ -86,38 +86,3 @@ func batchAffine*[M, N: static int, F](
        affs: var array[M, array[N, ECP_TwEdwards_Aff[F]]],
        projs: array[M, array[N, ECP_TwEdwards_Prj[F]]]) {.inline.} =
   batchAffine(affs[0].asUnchecked(), projs[0].asUnchecked(), M*N)
-
-func batchInvert*[F](
-        dst: ptr UncheckedArray[F],
-        elements: ptr UncheckedArray[F],
-        N: int
-      ) {.noInline.} =
-  ##  Montgomery's batch inversion
-  var zeros = allocStackArray(bool, N)
-  zeroMem(zeros, N)
-
-  var accumulator: F
-  accumulator.setOne()    # sets the accumulator to 1
-
-  for i in 0 ..< N: 
-    if elements[i].isZero().bool():
-      zeros[i] = true
-      continue
```

Comment:
This is missing a `dst[i].setZeros` to be consistent with single field inversion